GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Remotes.cs

```diff
@@ -313,20 +313,20 @@ protected override void ApplyStyle()
             {
                 base.ApplyStyle();
 
-                TreeViewNode.ToolTipText = _remote.FetchUrl != _remote.PushUrl
-                    ? $"Push: {_remote.PushUrl}\nFetch: {_remote.FetchUrl}"
+                TreeViewNode.ToolTipText = _remote.FetchUrl != _remote.PushUrls[0]
```

Comment:
Do you think it brings more meaning to introduce a property 'DefaultPushUrl' that do this job to select the first element (and also maybe a control that it exists)?

Reply:
This is the only situation where push url is currently used. If we are to change something it is probably to show all push url instead.

Reply:
I will propose changing to show all PushUrl in #7256 
Will just rebase before merge